devtools/conda-envs/examples_env.yml

```diff
@@ -24,3 +24,5 @@ dependencies:
     - jax
     - parmed<4
     - nglview>=3.0.6
+    - pip:
+        - git+https://github.com/openforcefield/openff-interchange.git@v0.3.25b1
```

Comment:
Can this be removed now that 0.3.25 has been released?

```suggestion
```

If you want a version of Interchange that's compatible with 0.3.25 installed with conda, but you want to take the examples from this tag, the best way to do that is to get `~=0.3.25` in the regular way and then set the tag in the `GITHUB_REPOS` global variable (l33-39 in `globals_.py`, change `"openforcefield/openff-interchange"` to `"openforcefield/openff-interchange@v0.3.25b1"`). I'd like to avoid pip installing things in this environment if possible.

Reply:
I just forgot to update this is all